module/Database/src/Model/SubDecision/OrganRegulation.php

```diff
@@ -0,0 +1,238 @@
+    {
+        return $this->version;
+    }
+
+    /**
+     * Set the version.
+     */
+    public function setVersion(string $version): void
+    {
+        $this->version = $version;
+    }
+
+    /**
+     * Get the date.
+     */
+    public function getDate(): DateTime
+    {
+        return $this->date;
+    }
+
+    /**
+     * Set the date.
+     */
+    public function setDate(DateTime $date): void
+    {
+        $this->date = $date;
+    }
+
+    /**
+     * Get approval status.
+     */
+    public function getApproval(): bool
+    {
+        return $this->approval;
+    }
+
+    /**
+     * Set approval status.
+     */
+    public function setApproval(bool $approval): void
+    {
+        $this->approval = $approval;
+    }
+
+    /**
+     * Get if changes were made.
+     */
+    public function getChanges(): bool
+    {
+        return $this->changes;
+    }
+
+    /**
+     * Set if changes were made.
+     */
+    public function setChanges(bool $changes): void
+    {
+        $this->changes = $changes;
+    }
+
+    /**
+     * Get the content.
+     */
+    public function getContent(): string
+    {
+        $template = $this->getTemplate();
+        $template = str_replace('%NAME%', $this->getName(), $template);
+        if (null === $this->getAuthor()) {
+            $template = str_replace('%AUTHOR%', 'onbekend', $template);
+        } else {
+            $template = str_replace('%AUTHOR%', $this->getAuthor()->getFullName(), $template);
+        }
+
+        $template = str_replace('%VERSION%', $this->getVersion(), $template);
+        $template = str_replace('%DATE%', $this->formatDate($this->getDate()), $template);
+        if ($this->getApproval()) {
+            $template = str_replace('%APPROVAL%', 'goedgekeurd', $template);
+            if ($this->getChanges()) {
+                $template = str_replace('%CHANGES%', ' met genoemde wijzigingen', $template);
+            } else {
+                $template = str_replace('%CHANGES%', '', $template);
+            }
+        } else {
+            $template = str_replace('%APPROVAL%', 'afgekeurd', $template);
+            $template = str_replace('%CHANGES%', '', $template);
+        }
+
+        return $template;
+    }
+
+    /**
+     * Format the date.
+     *
+     * returns the localized version of $date->format('d F Y')
+     *
+     * @return string Formatted date
+     */
+    protected function formatDate(DateTime $date): string
+    {
+        $formatter = new IntlDateFormatter(
+            'nl_NL', // yes, hardcoded :D
+            IntlDateFormatter::NONE,
+            IntlDateFormatter::NONE,
+            date_default_timezone_get(),
+            null,
+            'd MMMM Y',
+        );
+
+        return $formatter->format($date);
+    }
+
+    /**
+     * Decision template
+     */
+    protected function getTemplate(): string
+    {
+        return 'Het commissiereglement %NAME% van %AUTHOR%, versie %VERSION% van %DATE% wordt %APPROVAL%%CHANGES%.';
```

Comment:
My last comments are actually not entirely correct. This text is not valid for fraternities, which use `dispuutsreglement`. So there should actually be some kind of organ type association, but it can just be `committee` vs `fraternity` and not the whole list of types.

Reply:
I have changed it such that there is only the option of committee or fraternity, and included it in the decision template. It also correctly prints either commissiereglement or dispuut_s_reglement although I am not sure whether I implemented that optimally.